src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final int VALUE_INDEX = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] data = readFromFile(fromFileName);
+        String report = createReport(data);
+        writeToFile(toFileName, report);
+    }
+
+    private String createReport(String[] readedLine) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+
+        for (String s : readedLine) {
+            String[] tempArray = s.split(",");
+            for (int i = 0; i < tempArray.length; i++) {
+                if (tempArray[i].equals("supply")) {
+                    supply = supply + Integer.parseInt(tempArray[VALUE_INDEX]);
+                } else if (tempArray[i].equals("buy")) {
+                    buy = buy + Integer.parseInt(tempArray[VALUE_INDEX]);
+                }
+            }
```

Comment:
```suggestion
            if (tempArray[OPERATION_INDEX].equals("supply")) {
                supply += Integer.parseInt(tempArray[VALUE_INDEX]);
            } else if (tempArray[OPERATION_INDEX].equals("buy")) {
                buy += Integer.parseInt(tempArray[VALUE_INDEX]);
            }
```
lets create constants for operation index